capitains_nautilus/cts/collections.py

```diff
@@ -14,6 +14,7 @@
 from capitains_nautilus.collections.sparql import SparqlNavigatedCollection
 from rdflib import BNode, Literal, RDF, URIRef
 from capitains_nautilus.errors import UnknownCollection
+from capitains_nautilus.utils.performances import cached_property, STORE
 
 
 class CTSSparqlNavigatedCollection(PrototypeCtsCollection, SparqlNavigatedCollection):
```

Comment:
`CTSSparqlNavigatedCollection` should probably be `CtsSparqlNavigatedCollection`